src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from "react";
+import { DesktopNavbar } from "./DesktopNavbar";
+import { MobileNavbar } from "./MobileNavbar";
+
+function getWidth() {
+  return Math.max(
+    document.body.scrollWidth,
+    document.documentElement.scrollWidth,
+    document.body.offsetWidth,
+    document.documentElement.offsetWidth,
+    document.documentElement.clientWidth,
+  );
+}
+
+export const Navbar = () => {
+  const [width, setWidth] = useState(null);
+
+  const handleResize = () => [setWidth(getWidth())];
+
+  useEffect(() => {
+    setWidth(getWidth());
+    window.addEventListener("resize", handleResize);
+
+    return () => {
+      window.removeEventListener("resize", handleResize);
+    };
+  }, []);
+
+  if (width < 800) {
+    return <MobileNavbar />;
+  } else {
+    return <DesktopNavbar />;
+  }
+};
```

Comment:
We've got a `useMediaQuery` hook in a `hooks/` folder which would be cool if you could switch to? Also have some predefined breakpoints in `utils/breakpoints.ts` if you wanna set it to one of those, but I know you might need a different one depending on whether it looks nice with the predefined ones